hcloud/resource_hcloud_sshkey.go

```diff
@@ -179,3 +180,19 @@ func setSSHKeySchema(d *schema.ResourceData, s *hcloud.SSHKey) {
 	d.Set("public_key", s.PublicKey)
 	d.Set("labels", s.Labels)
 }
+
+func setSSHKeysSchema(d *schema.ResourceData, keys []*hcloud.SSHKey) {
+	var keyMaps []map[string]interface{}
+	for _, key := range keys {
+		keyMaps = append(keyMaps, map[string]interface{}{
+			"id":          key.ID,
+			"name":        key.Name,
+			"fingerprint": key.Fingerprint,
+			"public_key":  key.PublicKey,
+			"labels":      key.Labels,
+		})
+	}
+
+	d.SetId(time.Now().UTC().String())
```

Comment:
I don't know why but I guess I would not use a timestamp here, i will think about this and will let you know....

Reply:
Yeah I know what you mean, I wasn't sure of this either. How about concatenating all the IDs together with hyphens?

Reply:
I think this could be a good idea.